arrange_schedule.py

```diff
@@ -613,20 +613,15 @@ def edit_schedule(json_obj):
             display_time = int(display_time_list[num0])
             
             #get now sn
-            sql = ("SELECT sche_sn FROM schedule WHERE sche_is_used=0 ORDER BY sche_sn ASC LIMIT 1")
-            pure_result = db.query(sql)
-            if len(pure_result)>0:
+            with ScheduleDao() as scheduleDao:
+                target_sn = scheduleDao.getDisplayingSchedule()
+            if target_sn>0:
```

Comment:
The error of getDisplayingSchedule will return `None`, so this check should be `if target_sn:`